test/node/ws-api/ws.stop-propagation.test.ts

```diff
@@ -28,7 +28,7 @@ afterAll(async () => {
 })
 
 it('stops propagation for client "message" event', async () => {
-  const clientMessageListener = vi.fn<[number]>()
+  const clientMessageListener = vi.fn<(input: number) => void>()
```

Comment:
Is this change actually related to ESM migration?

Reply:
No but Vitest 2.0 has bugs related to executing ESM so a 3.0 bump is necessary. This is the only change that comes with that bump.